xarray/tests/test_concat.py

```diff
@@ -50,10 +52,10 @@ def test_concat_compat():
 
 class TestConcatDataset:
     @pytest.fixture
-    def data(self):
+    def data(self) -> None:
```

Comment:
```suggestion
    def data(self) -> Dataset:
```
Seems to return a Dataset, right?